services/tests/test_activation.py

```diff
@@ -89,11 +88,57 @@ def test_activate_user_success_for_enterprise_pr_billing(
         dbsession.commit()
         assert user.ownerid in org.plan_activated_users
 
+    def test_activate_user_success_user_org_overlap(
+        self, request, dbsession, mock_configuration, mocker, with_sql_functions
+    ):
+        mocker.patch("services.license.is_enterprise", return_value=True)
+        mocker.patch("services.license._get_now", return_value=datetime(2020, 4, 2))
+
+        # Create two orgs to ensure our seat availability checking works across
+        # multiple organizations.
+        org = OwnerFactory.create(
+            service="github",
+            oauth_token=None,
+            plan_activated_users=list(range(15, 20)),
+            plan_auto_activate=True,
+        )
+        dbsession.add(org)
+        dbsession.flush()
+
+        org_second = OwnerFactory.create(
+            service="github",
+            oauth_token=None,
+            plan_activated_users=list(
+                range(
+                    20,
+                    24,
+                )
+            ),
+            plan_auto_activate=True,
+        )
+        dbsession.add(org_second)
+        dbsession.flush()
+
+        encrypted_license = "wxWEJyYgIcFpi6nBSyKQZQeaQ9Eqpo3SXyUomAqQOzOFjdYB3A8fFM1rm+kOt2ehy9w95AzrQqrqfxi9HJIb2zLOMOB9tSy52OykVCzFtKPBNsXU/y5pQKOfV7iI3w9CHFh3tDwSwgjg8UsMXwQPOhrpvl2GdHpwEhFdaM2O3vY7iElFgZfk5D9E7qEnp+WysQwHKxDeKLI7jWCnBCBJLDjBJRSz0H7AfU55RQDqtTrnR+rsLDHOzJ80/VxwVYhb"
```

Comment:
How many seats does this license allows for?

Reply:
added a comment to the code